smartsim/_core/mli/infrastructure/worker/worker.py

```diff
@@ -191,6 +191,18 @@ def has_output_keys(self) -> bool:
         return self.output_keys is not None and bool(self.output_keys)
 
 
+@dataclass
+class TensorMeta:
+    """Metadata about a tensor, built from TensorDescriptors."""
+
+    dimensions: t.List[int]
+    """Dimensions of the tensor"""
+    order: str
+    """Order of the tensor"""
+    datatype: str
+    """Datatype of the tensor"""
```

Comment:
Ditto on this one. it seems hard to know what this string should be. I'm potentially suggesting something bad that makes it harder to keep the comment up to date, but this could have an example : foo.datatype = str(np.int32) or str(int) or ... anything to know what this string should really be

Reply:
Added more description! This datatype string is pulled from enums in `tensor_capnp` so I stated that and added examples.